src/Language/PureScript/Docs/Types.hs

```diff
@@ -413,6 +413,7 @@ asDeclarationInfo = do
     "typeClass" ->
       TypeClassDeclaration <$> key "arguments" asTypeArguments
                            <*> key "superclasses" (eachInArray asConstraint)
+                           <*> key "fundeps" asFunDeps
```

Comment:
This is a breaking change for the Pursuit format, right? Can this field be optional in the JSON?